backend/books/views.py

```diff
@@ -99,7 +99,22 @@ def create(self, request, *args, **kwargs):
         self.getOrCreateModel(request.data['authors'], Author)
         self.getOrCreateModel(request.data['genres'], Genre)
 
-        serializer = self.get_serializer(data=request.data)
+        # Handle the isbn that is already in DB
+        try:
+            obj = Book.objects.get(isbn_13=request.data['isbn_13'])
+        except Exception as e:
+            obj = None
+
+        # If the object with the specific isbn_13 is found we do the following:
+        # 1. add the isGhost field to the request data
+        # 2. update the already existing row in DB
+        if obj is not None:
+            request.data['isGhost'] = False
+            serializer = self.get_serializer(obj, data=request.data, partial=False)
+        else:
+            # This is different from the above serializer because this is creating a new row in the table
+            serializer = self.get_serializer(data=request.data)
```

Comment:
You allowed `isGhost` to be null in database table. The `request.data` won't have `isGhost` in it, is my understanding. Below on line 130 where you create the `default_query_set`, you filter on `isGhost=false`. Does this filter exclude all of the data where `isGhost` is null? If so, is this problematic since it appears on creation `isGhost` is null?

Reply:
So isGhost is not nullable as the default=False but I have deleted the null=True in models.py for clarity

Reply:
Later on I am going to refactor the Book model to separate ListView and CreateView for more clarity and cleanliness.
But I am not going to spend time on it now as it works but issue #144 addresses that